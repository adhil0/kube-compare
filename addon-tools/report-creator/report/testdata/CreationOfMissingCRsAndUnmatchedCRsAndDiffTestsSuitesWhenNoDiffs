{"Summary":{"RequiredCRS":{},"NumMissing":0,"UnmatchedCRS":[],"NumDiffCRs":0,"TotalCRs":11},"Diffs":[{"DiffOutput":"","CorrelatedTemplate":"cm.yaml","CRName":"v1_ConfigMap_kubernetes-dashboard_kubernetes-dashboard-settings"},{"DiffOutput":"","CorrelatedTemplate":"cr.yaml","CRName":"rbac.authorization.k8s.io/v1_ClusterRole_kubernetes-dashboard"},{"DiffOutput":"","CorrelatedTemplate":"crb.yaml","CRName":"rbac.authorization.k8s.io/v1_ClusterRoleBinding_kubernetes-dashboard"},{"DiffOutput":"","CorrelatedTemplate":"deploymentDashboard.yaml","CRName":"apps/v1_Deployment_kubernetes-dashboard_kubernetes-dashboard"},{"DiffOutput":"","CorrelatedTemplate":"deploymentMetrics.yaml","CRName":"apps/v1_Deployment_kubernetes-dashboard_dashboard-metrics-scraper"},{"DiffOutput":"","CorrelatedTemplate":"ns.yaml","CRName":"v1_Namespace_kubernetes-dashboard"},{"DiffOutput":"","CorrelatedTemplate":"rb.yaml","CRName":"rbac.authorization.k8s.io/v1_RoleBinding_kubernetes-dashboard_kubernetes-dashboard"},{"DiffOutput":"","CorrelatedTemplate":"role.yaml","CRName":"rbac.authorization.k8s.io/v1_Role_kubernetes-dashboard_kubernetes-dashboard"},{"DiffOutput":"","CorrelatedTemplate":"sa.yaml","CRName":"v1_ServiceAccount_kubernetes-dashboard_kubernetes-dashboard"},{"DiffOutput":"","CorrelatedTemplate":"secret.yaml","CRName":"v1_Secret_kubernetes-dashboard_kubernetes-dashboard-certs"},{"DiffOutput":"","CorrelatedTemplate":"secret.yaml","CRName":"v1_Secret_kubernetes-dashboard_kubernetes-dashboard-csrf"},{"DiffOutput":"","CorrelatedTemplate":"secret.yaml","CRName":"v1_Secret_kubernetes-dashboard_kubernetes-dashboard-key-holder"},{"DiffOutput":"","CorrelatedTemplate":"service.yaml","CRName":"v1_Service_kubernetes-dashboard_kubernetes-dashboard"},{"DiffOutput":"","CorrelatedTemplate":"ns.yaml","CRName":"v1_Namespace_kubernetes-dashboard"},{"DiffOutput":"","CorrelatedTemplate":"sa.yaml","CRName":"v1_ServiceAccount_kubernetes-dashboard_kubernetes-dashboard"},{"DiffOutput":"","CorrelatedTemplate":"service.yaml","CRName":"v1_Service_kubernetes-dashboard_kubernetes-dashboard"},{"DiffOutput":"","CorrelatedTemplate":"secret.yaml","CRName":"v1_Secret_kubernetes-dashboard_kubernetes-dashboard-certs"},{"DiffOutput":"","CorrelatedTemplate":"secret.yaml","CRName":"v1_Secret_kubernetes-dashboard_kubernetes-dashboard-csrf"},{"DiffOutput":"","CorrelatedTemplate":"secret.yaml","CRName":"v1_Secret_kubernetes-dashboard_kubernetes-dashboard-key-holder"},{"DiffOutput":"","CorrelatedTemplate":"cm.yaml","CRName":"v1_ConfigMap_kubernetes-dashboard_kubernetes-dashboard-settings"},{"DiffOutput":"","CorrelatedTemplate":"role.yaml","CRName":"rbac.authorization.k8s.io/v1_Role_kubernetes-dashboard_kubernetes-dashboard"},{"DiffOutput":"","CorrelatedTemplate":"cr.yaml","CRName":"rbac.authorization.k8s.io/v1_ClusterRole_kubernetes-dashboard"},{"DiffOutput":"","CorrelatedTemplate":"rb.yaml","CRName":"rbac.authorization.k8s.io/v1_RoleBinding_kubernetes-dashboard_kubernetes-dashboard"},{"DiffOutput":"","CorrelatedTemplate":"crb.yaml","CRName":"rbac.authorization.k8s.io/v1_ClusterRoleBinding_kubernetes-dashboard"},{"DiffOutput":"","CorrelatedTemplate":"deploymentDashboard.yaml","CRName":"apps/v1_Deployment_kubernetes-dashboard_kubernetes-dashboard"},{"DiffOutput":"","CorrelatedTemplate":"service.yaml","CRName":"v1_Service_kubernetes-dashboard_dashboard-metrics-scraper"},{"DiffOutput":"","CorrelatedTemplate":"deploymentMetrics.yaml","CRName":"apps/v1_Deployment_kubernetes-dashboard_dashboard-metrics-scraper"}]}
