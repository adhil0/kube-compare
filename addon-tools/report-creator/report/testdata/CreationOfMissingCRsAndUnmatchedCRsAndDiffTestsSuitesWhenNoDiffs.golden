<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Comparison results of known valid reference configuration and a set of specific cluster CRs" tests="29" failures="0" errors="0" TIME>
	<testsuite tests="27" failures="0" TIME name="Detected Differences Between Cluster CRs and Expected CRs" TIME>
		<properties></properties>
		<testcase classname="Matching Reference CR: cm.yaml" name="CR: v1_ConfigMap_kubernetes-dashboard_kubernetes-dashboard-settings" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: cr.yaml" name="CR: rbac.authorization.k8s.io/v1_ClusterRole_kubernetes-dashboard" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: crb.yaml" name="CR: rbac.authorization.k8s.io/v1_ClusterRoleBinding_kubernetes-dashboard" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: deploymentDashboard.yaml" name="CR: apps/v1_Deployment_kubernetes-dashboard_kubernetes-dashboard" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: deploymentMetrics.yaml" name="CR: apps/v1_Deployment_kubernetes-dashboard_dashboard-metrics-scraper" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: ns.yaml" name="CR: v1_Namespace_kubernetes-dashboard" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: rb.yaml" name="CR: rbac.authorization.k8s.io/v1_RoleBinding_kubernetes-dashboard_kubernetes-dashboard" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: role.yaml" name="CR: rbac.authorization.k8s.io/v1_Role_kubernetes-dashboard_kubernetes-dashboard" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: sa.yaml" name="CR: v1_ServiceAccount_kubernetes-dashboard_kubernetes-dashboard" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: secret.yaml" name="CR: v1_Secret_kubernetes-dashboard_kubernetes-dashboard-certs" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: secret.yaml" name="CR: v1_Secret_kubernetes-dashboard_kubernetes-dashboard-csrf" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: secret.yaml" name="CR: v1_Secret_kubernetes-dashboard_kubernetes-dashboard-key-holder" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: service.yaml" name="CR: v1_Service_kubernetes-dashboard_kubernetes-dashboard" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: ns.yaml" name="CR: v1_Namespace_kubernetes-dashboard" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: sa.yaml" name="CR: v1_ServiceAccount_kubernetes-dashboard_kubernetes-dashboard" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: service.yaml" name="CR: v1_Service_kubernetes-dashboard_kubernetes-dashboard" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: secret.yaml" name="CR: v1_Secret_kubernetes-dashboard_kubernetes-dashboard-certs" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: secret.yaml" name="CR: v1_Secret_kubernetes-dashboard_kubernetes-dashboard-csrf" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: secret.yaml" name="CR: v1_Secret_kubernetes-dashboard_kubernetes-dashboard-key-holder" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: cm.yaml" name="CR: v1_ConfigMap_kubernetes-dashboard_kubernetes-dashboard-settings" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: role.yaml" name="CR: rbac.authorization.k8s.io/v1_Role_kubernetes-dashboard_kubernetes-dashboard" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: cr.yaml" name="CR: rbac.authorization.k8s.io/v1_ClusterRole_kubernetes-dashboard" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: rb.yaml" name="CR: rbac.authorization.k8s.io/v1_RoleBinding_kubernetes-dashboard_kubernetes-dashboard" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: crb.yaml" name="CR: rbac.authorization.k8s.io/v1_ClusterRoleBinding_kubernetes-dashboard" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: deploymentDashboard.yaml" name="CR: apps/v1_Deployment_kubernetes-dashboard_kubernetes-dashboard" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: service.yaml" name="CR: v1_Service_kubernetes-dashboard_dashboard-metrics-scraper" TIME>
			<properties></properties>
		</testcase>
		<testcase classname="Matching Reference CR: deploymentMetrics.yaml" name="CR: apps/v1_Deployment_kubernetes-dashboard_dashboard-metrics-scraper" TIME>
			<properties></properties>
		</testcase>
	</testsuite>
	<testsuite tests="1" failures="0" TIME name="Reference validation" TIME>
		<properties></properties>
		<testcase classname="" name="No reference validation issues: All expected CRs exist in the cluster" TIME>
			<properties></properties>
		</testcase>
	</testsuite>
	<testsuite tests="1" failures="0" TIME name="Unmatched Cluster Resources" TIME>
		<properties></properties>
		<testcase classname="" name="All Cluster CRs are matched to reference CRs " TIME>
			<properties></properties>
		</testcase>
	</testsuite>
</testsuites>